composer.json

```diff
@@ -35,18 +35,25 @@
 		"ext-dom": "*",
 		"ext-filter": "*",
 		"composer/installers": "^1.0.12",
-		"mediawiki/mediawiki": ">=1.31",
```

Comment:
This cannot be removed in Chameleon 2.x

Reply:
Sorry, I needed to remove it temporarily to get PHP code sniffer to work from composer. I meant to add it back in (until the changes for 3.x) and will do so now.